### ISSUE TITLE
[Bug]: Display Name for properties is not displayed on the old mouseover stat sheet

### ISSUE THREAD

**Issue by Pmofmalasia** [RPTools/maptool#4636]
### Describe the Bug

Title says it all

### To Reproduce

Make property, give it display name, show on stat sheet, mouse over token

### Expected Behaviour

Display name shows on stat sheet

**Comment by bubblobill** [comment]
Correct. Old stat-sheet works on the old properties that had no display name.
Not a bug. More an expectation not met.

**Comment by Pmofmalasia** [comment]
Maybe mislabeled, but I put in the issue because Craig requested it.
